### ISSUE TITLE
dub init: Append to .gitignore instead of bailing

### ISSUE THREAD

**Issue by Wyatts** [dlang/dub#758]
It's a papercut to have to move the existing one, run dub init, then copy the old stuff over.

If you want to be polite, maybe ask the user if its okay to stick stuff in there when `'.gitignore'.exists`.

**Comment by Geod24** [comment]
Recently hit that myself as well. We might want to copy git behaviour here, where it is okay to `git init` an existing repository, [as said in the man](https://www.kernel.org/pub/software/scm/git/docs/git-init.html):

```
Running git init in an existing repository is safe. It will not overwrite things that are already there. The primary reason for rerunning git init is to pick up newly added templates (or to move the repository to another place if --separate-git-dir is given).
```

**Comment by jacob-carlborg** [comment]
👍

**Comment by Geod24** [comment]
Should we close this ?

**Comment by Geod24** [comment]
Solved by #1330 (not the solution prescribed here, but it solves the issue)
